### PR TITLE
power ibm z updates

### PR DESCRIPTION
**Pull request by SNiemann15**

### CODE REVIEW

File: modules/installation-requirements-user-infra.adoc [hunk 1]
```diff
@@ -181,6 +181,7 @@ On your z/VM instances, set up:
 * 3 guest virtual machines for {product-title} control plane machines, one per z/VM instance
 * At least 6 guest virtual machines for {product-title} compute machines, distributed across the z/VM instances
 * 1 guest virtual machine for the temporary {product-title} bootstrap machine
+* To ensure the availability of integral components in an overcommitted environment, increase the priority of the control plane using the cp command SET SHARE. Do the same for infrastructure plane machines if they exist. See link:https://www.ibm.com/support/knowledgecenter/en/SSB27U_7.1.0/com.ibm.zvm.v710.hcpb7/setshare.htm[SET SHARE] in the IBM Knowledge Center.
```
Comment (jeana-redhat) [inline modules/installation-requirements-user-infra.adoc:184]: If SET SHARE is a command, it should be enclosed in backticks (``):
> using the cp command `SET SHARE`.

Also, on the page you've linked, they capitalize as "CP commands" so maybe we should here as well?

**Review by jeana-redhat — COMMENTED** [review]
A few suggestions, overall looking good :+1:

File: installing/installing_ibm_power/installing-ibm-power.adoc [hunk 2]
```diff
@@ -1,17 +1,17 @@
 [id="installing-ibm-power"]
-= Installing a cluster on IBM Power
+= Installing a cluster on IBM Power Systems
```
[the remark this answers is not in the document]
Comment (SNiemann15) [reply installing/installing_ibm_power/installing-ibm-power.adoc:2]: Yes you are correct. Where do I change the topic map?

[the remark this answers is not in the document]
Comment (jeana-redhat) [reply installing/installing_ibm_power/installing-ibm-power.adoc:2]: It's this file here: https://github.com/openshift/openshift-docs/blob/master/_topic_map.yml

**Comment by holgwolf** [comment]
QE review done for IBM Z updates lgtm

**Comment by SNiemann15** [comment]
@jeana-redhat Hi Joanna I made the requested changes please have a look. Thanks Silke

**Comment by jeana-redhat** [comment]
@SNiemann15 looking good! Did you also want to get this part to say "System"?

IBMPowerChapter

**Comment by SNiemann15** [comment]
@jeana-redhat Damn I missed that one will change it on Monday.

**Comment by SNiemann15** [comment]
@jeana-redhat Done. I hope that was the last one

**Comment by jeana-redhat** [comment]
lgtm! do you need me to merge?

**Pull request merged**